City-of-Helsinki/drupal-helfi-platform-config
# UHF-6993: Update cookie policy version.

# [UHF-6993](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6993)
Update cookie policy version.

## What was done
* Update hook to change cookie policy version.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git checkout dev`
    * `git pull`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-6993-update-cookie-policy-version`
* Run `make drush-updb drush-cr`

## How to test
* While looking at the cookies for cookie compliance, observe the version changes after update and the banner reappears once when the version is updated.

## Other PRs
*
